thirdeye-worker/src/main/java/ai/startree/thirdeye/worker/task/runner/DetectionPipelineTaskRunner.java

```diff
@@ -183,12 +190,12 @@ private void postExecution(final OperatorResult result) {
     if (anomalies == null) {
       return;
     }
-    for (final AnomalyDTO anomalyDTO : anomalies) {
+    anomalies.stream().map(this::setAnomalyAuth).forEach(anomalyDTO -> {
       final Long id = anomalyDao.save(anomalyDTO);
```

Comment:
nitpick: in fluent apis, I try to only use methodReference 
`map(this::doThis)` `map(ClassName::doThisStatic)` or one liners `map(elem -> doThat(elem, 4))`

so in this case it could look like: 
```
anomalies.stream()
  .map(this::setAnomalyAuth)
  .forEach(this::saveAnomaly)
```
the mix of fluent and imperative tends to be less readable imo.
but here it's the limit case, both ways are not that good (the extracted method is shallow)

--> your call.